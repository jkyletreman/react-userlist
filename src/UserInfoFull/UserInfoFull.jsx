import React from "react";

const UserInfoFull = ({ additionalInfo }) => {
  console.log(additionalInfo);
  return (
    <ul>
      <li>k</li>
    </ul>
  );
};

export default UserInfoFull;
