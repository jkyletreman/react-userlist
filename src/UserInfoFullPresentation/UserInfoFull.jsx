import React from "react";
import UserData from "../context";

const UserInfoFull = birthYear => {
  return (
    <UserData.Consumer>
      {user => (
        console.log(user)

      )}
    </UserData.Consumer>
  );
};

export default UserInfoFull;

{/* <ul>
  <li>{user.first}</li>
  <li>{user.last}</li>
  <li>{birthYear}</li>
  <li>{user.location}</li>
  <li>{user.description}</li>
</ul> */}
