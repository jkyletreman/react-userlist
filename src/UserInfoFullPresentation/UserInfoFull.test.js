import React from "react";
import ReactDOM from "react-dom";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";
import UserInfoFull from "./UserInfoFull";

configure({ adapter: new Adapter() });

describe("<UserInfoFull />", () => {
  it("renders shallow correctly", () => {
    const tree = shallow(
      <UserInfoFull
        birthYear={1989}
        selectedUser={{
          first: "kyle",
          last: "treman",
          location: "NY",
          description: "working"
        }}
      />
    );
    expect(toJson(tree)).toMatchSnapshot();
  });
});
