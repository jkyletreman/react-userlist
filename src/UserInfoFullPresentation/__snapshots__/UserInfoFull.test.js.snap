// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserInfoFull /> renders shallow correctly 1`] = `
<UserInfoFull
  birthYear={1989}
  selectedUser={
    Object {
      "description": "working",
      "first": "kyle",
      "last": "treman",
      "location": "NY",
    }
  }
>
  <ul>
    <li>
      kyle
    </li>
    <li>
      treman
    </li>
    <li>
      1989
    </li>
    <li>
      NY
    </li>
    <li>
      working
    </li>
  </ul>
</UserInfoFull>
`;
